docs/widgetbook-cloud/integrating-github.mdx

```diff
@@ -0,0 +1,57 @@
+# Integrating GitHub App with Widgetbook Cloud
+
+A GitHub App is a first-class actor on GitHub, meaning it has its own identity separate from users.
+GitHub Apps can perform tasks such as sending messages or opening issues, and they can be installed directly on organizations and user accounts and granted access to specific repositories. They offer granular permissions, enhancing security by limiting access to only what's necessary.
+
+To streamline your workflow and improve your overall efficiency, you can integrate GitHub App with Widgetbook Cloud.
+
+<video width="100%" height="720px" autoplay muted loop>
+  <source src="/assets/integrating-github/github-integration-hd.mp4" type="video/mp4" />
+  Your browser does not support the video tag.
+</video>
+
+Here's a step-by-step guide on how to go about it:
+
+## Step 1: Initiate via your Company Profile
+
+First, navigate to your company's profile on Widgetbook Cloud. This can be done by selecting 'Company' from the navigation rail on your dashboard.
+
+<Image src="/assets/integrating-github/1.png" zoom />
+
+## Step 2: Select your Company
+
+Select the company you want to integrate from the list of available companies with the GitHub App.
+
+<Image src="/assets/integrating-github/2.png" zoom />
+
+## Step 3: Start GitHub App Installation Flow
+
+Now, you're ready to start the GitHub App installation process. This process requires administrative permissions for the repository. If you lack these permissions, ask your repository admin to initiate the process.
+
+<Image src="/assets/integrating-github/3.png" zoom />
+
+## Step 4: Choose your Repositories
+
+During the installation process, you can select the repositories you want to integrate with Widgetbook Cloud. Choose the ones that best fit your needs.
+
+## Step 5: Link to a Project
+
+After selecting the repositories, choose a project that has already been linked to your selected company.
+
+<Image src="/assets/integrating-github/5.png" zoom />
+
+## Step 6: Navigate to Settings
+
+Once your project is linked, navigate to the settings menu. This is where you'll finalize the connection between your repository and Widgetbook Cloud.
+
+## Step 7: Connect your Repository
+
+Under the "Code repository" section in settings, connect and select the repository you want to use. This links your GitHub App and the chosen repository to Widgetbook Cloud.
+
+<Image src="/assets/integrating-github/7.png" zoom />
+
+## Step 8: Sync Reviews with Pull Requests
+
+Widgetbook Cloud reviews will be automatically generated and synchronized with each pull request after completing these steps. This streamlines your review process and helps keep everything organized.
+
```

Comment:
Do you have a Widgetbook project running? If so, can you add a screenshot of the review page here? 
![Screenshot 2023-05-23 at 11 33 34](https://github.com/widgetbook/widgetbook/assets/69841305/5ad7b5b7-e521-48e2-b6fd-f2ca089e0900)

In this demo, there is no open review, so I don't want to use this screenshot. If you have one with an open review, please use it. Otherwise let me know and I create an open review in this demo project and send you screenshot

Reply:
I added review page